code/modules/dungeons/teleporter.dm

```diff
@@ -150,6 +150,11 @@
 	sleep(100)
 	update_icon()
 
+	for(var/mob/living/simple_animal/A in target.loc.loc)
+		spawn(1)
```

Comment:
It's worth it to check is `A` still exist/not qdeled.

Reply:
Okay, but, where would they go anyway?